tests/tt_metal/tt_metal/unit_tests/ethernet/basic_eth_kernels.cpp

```diff
@@ -710,3 +824,56 @@ TEST_F(N300DeviceFixture, EthKernelsRandomEthPacketSizeDirectSendTests) {
         }
     }
 }
+
+TEST_F(BlackholeSingleCardFixture, EthKernelOnIdleErisc0) {
```

Comment:
I see the test below uses riscv_1, does this test run on BH only?
add a comment above each test to state what the test tets?

Reply:
yeah it only runs on BH, `BlackholeSingleCardFixture` will skip over non-BH devices

Reply:
should we run the erisc0 test on WH as well?